tribler.sh

```diff
@@ -40,6 +40,9 @@ else
             exit 1
         fi
 
+	PYTHONPATH=.:"$PYTHONPATH"
```

Comment:
If it has to be done on both Linux and OSX, just move the Linux one out of the `if`.